kotest/src/test/kotlin/com/github/erikhuizinga/mockk/kotest/example/ExampleTestSpec.kt

```diff
@@ -0,0 +1,29 @@
+package com.github.erikhuizinga.mockk.kotest.example
+
+import com.github.erikhuizinga.mockk.kotest.MockkTestListener
+import io.kotlintest.shouldBe
+import io.kotlintest.specs.StringSpec
+import io.mockk.every
+import io.mockk.impl.annotations.RelaxedMockK
+
+class Client {
+    fun get() = "example"
+}
+
+class ExampleTestSpec : StringSpec() {
```

Comment:
This example should show the added value of `MockkTestListener`, which is in the fact that between tests the mock's state is cleared and after the test the mock is unmocked. Take a look at the JUnit 5 example for a way to show this. If you think of a shorter, more compact way, then that's very much welcome!